package.json

```diff
@@ -60,12 +58,14 @@
     "eslint-plugin-react": "^7.29.3",
     "eslint-plugin-react-hooks": "^4.3.0",
     "husky": "^7.0.4",
-    "jest": "^26.6.3",
+    "jest": "^27.5.1",
     "jest-expo": "~44.0.1",
-    "prettier": "^2.5.1",
+    "prettier": "2.5.1",
```

Comment:
Not really sure why prettier version here changes from `^2.5.1` meaning at least 2.5.1 to pinning the prettier version to be always `2.5.1`. I don't think we should do that, since by doing so we are cutting ourselves off prettier further updates and bugfixes

Reply:
Honestly, I don't know what happened here; changed it back :)